src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -119,6 +120,27 @@ public void execute_duplicatePersonFilteredList_failure() {
         assertCommandFailure(editCommand, model, EditCommand.MESSAGE_DUPLICATE_PERSON);
     }
 
+    @Test
+    public void execute_duplicateFieldsUnfilteredList_nameMustBeDifferent() {
```

Comment:
Quick comment: This three-part test naming format has a convention for what each part should be.

Reply:
I had troubles figuring out the name too. I was testing the execute method with two edits to represent the bug, first editing a person with similar fields(success), and then editing to a similar name(failure). Would it be better if it was `execute_differentNameDuplicateFieldsUnfilteredList_success()`? Or maybe this test case isn't necessary since we do not need to test on the non-transitive nature anymore.

Reply:
If you are testing both success and failure outcomes, you can omit the third part of the name.
Yes, this test case might be redundant.

Reply:
Thanks for the tip, I have removed the test case and updated this commit.